packages/angular-material/tsconfig.example.json

```diff
@@ -1,16 +1,13 @@
 {
-  "extends": "../../tsconfig.base",
+  "extends": "./tsconfig",
   "compilerOptions": {
-    "outDir": "./example/dist",
-    "sourceMap": true,
-    "target": "es5"
+    "outDir": "./example/dist"
   },
-  "exclude": ["node_modules"],
   "files": ["./example/main.ts"],
   "angularCompilerOptions": {
-    "compilationMode": "partial",
+    "compilationMode": "full",
```

Comment:
`"compilationMode": "full",` is the default value. Thus, we can just delete this line completely :)

Reply:
You're right!

Reply:
Strangely enough, if I remove it, it doesn't seem to compile in "full" mode even though the docs states it's the default value...

Reply:
strange but alright. then we'll leave it in :D